### ISSUE TITLE
Time only option

### ISSUE THREAD

**Issue by weedkiller** [uxsolutions/bootstrap-datepicker#2161]
Hi, is there a way to configure this to enter / validate only time, `text input + step options + control` 

i.e. let user Enter via - type/select/or step `hours` and `mins` option - and show place holder

I saw other controls, but do not want to load another lib, and would prefer to use your lib for all time related options.

thanks

**Comment by haferje** [comment]
I also wouldn't mind a way to enter both the date and time.

**Comment by haferje** [comment]
sorry, i too quickly jumped on this issue without doing my research...

@weedkiller, this addition has been requested many times before.  the latest comment on the subject i found seems to suggest it could be added to version 2 or 3 - it's on the roadmap, but not finalized when it will be implemented.

back at issue #29 there are many other library suggestions as replacements and #909 contained a potential pull request that never seemed to make it into the code and now seems too old of code to easily add it back in with the current supported features.

in summary, our options are to wait for v2 or v3, or choose a competing library.

**Comment by acrobat** [comment]
Time support won't come in v1. If someone is willing to work on this and we are willing to support this it could come in v2 but currently there are no exact plans.

If you are looking for time support you can use [Eonasdan/bootstrap-datetimepicker](https://github.com/Eonasdan/bootstrap-datetimepicker)
